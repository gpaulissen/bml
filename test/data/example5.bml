1C-(1D)-
D  4+!h
1H 4+!s
1S INV+ with 5+!d / Negative NT
1N Game forcing, 5+!c or balanced
2C 5+!c, 5--9 hcp
2X 6+ suit, 4--8 hcp
2N Invitational
