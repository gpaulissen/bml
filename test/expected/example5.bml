#VUL 00

#SEAT 0

1C
  (1D)
    D = 4+!h
    1H = 4+!s
    1S = INV+ with 5+!d / Negative NT
    1N = Game forcing, 5+!c or balanced
    2C = 5+!c, 5--9 hcp
    2H = 6+ suit, 4--8 hcp
    2S = 6+ suit, 4--8 hcp
    2N = Invitational
