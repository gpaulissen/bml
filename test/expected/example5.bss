*00{}=NYYYYYY
001C=NYYYYYY008
001C1D=NYYYYYY008
001C1DD=NYYYYYY04+!h
001C1D1H=NYYYYYY0084+!s
001C1D1S=NYYYYYY008INV+ with 5+!d / Negative NT
001C1D1N=NYYYYYY0Game forcing, 5+!c or balanced
001C1D2C=NYYYYYY0085+!c, 5--9 hcp
001C1D2H=NYYYYYY0086+ suit, 4--8 hcp
001C1D2S=NYYYYYY0086+ suit, 4--8 hcp
001C1D2N=NYYYYYY0Invitational
