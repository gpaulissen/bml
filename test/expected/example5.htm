<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC '-//W3C//DTD XHTML 1.0 Strict//EN' 'http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd'>
<html xmlns='http://www.w3.org/1999/xhtml' xml:lang='en' lang='en'><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><title>No title supplied</title><style type="text/css">div.start {
    display: inline-block;
    vertical-align: top;
    width: 60px;
}
    
div.bidtable {
    width:700px;
    padding-bottom: 20px;
}    

div.bidtable ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
}

div.bidtable li ul
{
    list-style-type: none;
    margin: 0;
    padding: 0;
    margin-left: 60px;
}

div.tree {
    list-style-type: none;
    padding-bottom: 20px;
}

div.tree ul {
    list-style-type: none;
    /* vline.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAKAQMAAABPHKYJAAAAA1BMVEWIiIhYZW6zAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH1ggGExMZBky19AAAAAtJREFUCNdjYMAEAAAUAAHlhrBKAAAAAElFTkSuQmCC) repeat-y;
    margin: 0;
    padding: 0;
    margin-left: 10px;
}

div.tree li.node {
    margin: 0;
    padding: 0 12px;
    line-height: 20px;
    /* node.png */
    background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIJYVaFGwAAAARSURBVAjXY2hgQIf/GTDFGgDSkwqATqpCHAAAAABJRU5ErkJggg==) no-repeat;
}

div.tree td.node {
    vertical-align: top;
    width: 60px;
}

div.tree li.last {
    /* lastnode.png */
    background: #fff url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAgAAAAUAQMAAACK1e4oAAAABlBMVEUAAwCIiIgd2JB2AAAAAXRSTlMAQObYZgAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9YIBhQIIhs+gc8AAAAQSURBVAjXY2hgQIf/GbAAAKCTBYBUjWvCAAAAAElFTkSuQmCC) no-repeat;
}

div.tree li.root {
    background: #fff none no-repeat;
}

.ccolor{color:Green}
.dcolor{color:Orange}
.hcolor{color:Red}
.scolor{color:Black}
</style></head><body><div class="bidtable"><ul><li><div class="start">1<span class="ccolor">&clubs;</span>-(1<span class="dcolor">&diams;</span>)</div><ul><li><div class="start">Dbl</div>4+<span class="hcolor">&hearts;</span></li><li><div class="start">1<span class="hcolor">&hearts;</span></div>4+<span class="scolor">&spades;</span></li><li><div class="start">1<span class="scolor">&spades;</span></div>INV+ with 5+<span class="dcolor">&diams;</span> / Negative NT</li><li><div class="start">1NT</div>Game forcing, 5+<span class="ccolor">&clubs;</span> or balanced</li><li><div class="start">2<span class="ccolor">&clubs;</span></div>5+<span class="ccolor">&clubs;</span>, 5&ndash;9 hcp</li><li><div class="start">2X</div>6+ suit, 4&ndash;8 hcp</li><li><div class="start">2NT</div>Invitational</li></ul></li></ul></div></body></html>